packages/fhir-converter/src/templates/cda/Resources/Condition.hbs

```diff
@@ -10,15 +10,14 @@
             {{/each}}
         ],
         "clinicalStatus":
-        [ 
             {{#each (toArray conditionEntry.entryRelationship)}}
                 {{#each (toArray this.observation.templateId)}}
                     {{#if (eq this.root '2.16.840.1.113883.10.20.22.4.6')}} 
                         {{>DataType/CodeableConcept.hbs code=../this.observation.value}},
                     {{/if}}
                 {{/each}}
             {{/each}}
-        ],  
+        ,  
```

Comment:
@Orta21 is this `,` necessary if it's in `{{>DataType/CodeableConcept.hbs code=../this.observation.value}},` ?